opentreemap/treemap/instance.py

```diff
@@ -449,6 +451,22 @@ def is_accessible_by(self, user):
         except ObjectDoesNotExist:
             return False
 
+    def export_enabled_for(self, user):
```

Comment:
I think it would make a little more sense to have `def export_enabled_for(instance, user)` in the `exporter/` app, for encapsulation. If still necessary, this method could import and thinly wrap it. This technique would have consistency with how the `EXPORTS_NOT_ENABLED_CONTEXT` dict is used.